app/assets/javascripts/calendars/guiders-appointments.es6

```diff
@@ -9,7 +9,13 @@
         defaultView: 'agendaDay',
         resourceLabelText: 'Guiders',
         header: {
-          right: 'agendaDay timelineDay today jumpToDate prev,next'
+          right: 'filter agendaDay timelineDay today jumpToDate prev,next'
+        },
+        customButtons: {
+          filter: {
+            text: 'Filter',
```

Comment:
`glyphicon glyphicon-filter` worth it or no?

Could potentially use these for the horizontal/vertical ones if we wanted to go crazy

<img width="212" alt="screen shot 2016-11-14 at 14 11 02" src="https://cloud.githubusercontent.com/assets/295469/20267783/35ee27c6-aa74-11e6-857f-012bc7c269cc.png">